napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -57,6 +98,36 @@ def __init__(self, layer: Layer) -> None:
 
         self.setLayout(LayerFormLayout(self))
 
+        # Buttons
+        self.button_group = QButtonGroup(self)
+        self.panzoom_button = self._radio_button(
+            layer,
+            'pan_zoom',
```

Comment:
```suggestion
            'pan',
```
Keep the pan icon, even though the mode is still called pan_zoom.